src/tt.cpp

```diff
@@ -66,8 +117,50 @@ uint8_t TTEntry::relative_age(const uint8_t generation8) const {
     // the result) to calculate the entry age correctly even after
     // generation8 overflows into the next cycle.
 
-    return (TranspositionTable::GENERATION_CYCLE + generation8 - genBound8)
-         & TranspositionTable::GENERATION_MASK;
+    return (GENERATION_CYCLE + generation8 - genBound8)
+         & GENERATION_MASK;
+}
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
no change in documentation style is needed

Reply:
Double lines between functions yea?

Reply:
Eh that is inconsistent across the code base